content/en/internal/configuration-database/access/_index.md

```diff
@@ -1,64 +1,65 @@
-description: "Accessing GOCDB"
-weight: 20
+description: "Accessing the Configuration Database"
+weight: 25
 type: "docs"
 ---
 
-The GOCDB can be accessed at [https://goc.egi.eu](https://goc.egi.eu).
+To access [web interface](https://goc.egi.eu) of the EGI Configuration Database
+(GOCDB), users can either:
 
-To access the web interface, users can:
+- [Use EGI Check-in](#using-institutional-account-via-egi-check-in)
+  with an institutional account, or
+- [Use an X.509 digital certificate](#using-an-x509-digital-certificate)
+  installed in the browser.
 
-- use EGI Check-in to [access the GOCDB with an institutional
-  account](#using-institutional-account-via-egi-check-in).
-- use an [X.509 digital certificate installed in the
-  browser](#using-an-x509-digital-certificate), delivered by
-  one of the recognised EU-Grid-PMA Certification Authorities ;
-
-Users can access the system as soon as they are authenticated however they will
-only be able to update information based on their roles.
+Users can access the system as soon as they are authenticated. However,
+they will only be able to update information based on their roles.
 More information about roles and associated permission is available in the
-[Users and
-roles](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
-section.
+[_Users and roles_](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
+section of the documentation.
 
-All roles applications need to be validated by parent roles or administrators.
-Once this is done, you can access/modify relevant information according to the
-role you have been granted. You can learn more on roles and user accounts by
-reading the #Users and roles section of this documentation.
+Applications requesting a specific role have to be validated by parent roles
+or administrators. Once granted, users can access and/or modify relevant
+information, according to the roles granted to the user.
 
 ## Using institutional account via EGI Check-in
 
 In order to be able to access the GOCDB with their institutional account, users
 need to:
 
 1. Have their Identity Provider (IdP) federated in EGI Check-in (via
-   [eduGAIN](https://edugain.org/) or directly) ;
+   [eduGAIN](https://edugain.org/) or directly)
 1. Have created an [EGI Check-in
-   account](https://wiki.egi.eu/wiki/AAI_usage_guide) ;
+   account](https://wiki.egi.eu/wiki/AAI_usage_guide)
 
 {{% alert title="Important" color="warning" %}}
 In the case the user cannot use an IdP compliant with [REFEDS
 R&S](https://refeds.org/research-and-scholarship) and [REFEDS
 Sirtfi](https://refeds.org/sirtfi), the user will have to request joining a
-specific group. Using a compliant IdP is the preferable solution.
-{{% /alert %}}
+specific group, by performing the steps below. Using a compliant IdP is the
+preferable solution.
 
 1. User should ask to join the [GOCDB user
-   group](https://aai.egi.eu/registry/co_petitions/start/coef:41) ;
```

Comment:
Not sure about how to handle punctuation in lists, we probably need a native speaker (@mviljoen-egi?) to help here.

Reply:
I am not using punctuation at end of list items, unless items contain more than one sentence (and then I add punctuation to all items). Please advise.

Reply:
Please follow those rules: https://docs.openstack.org/doc-contrib-guide/writing-style/lists.html

Reply:
I am already compliant with list punctuations in most of the content that I wrote. As this is a numbered list, I added dots.